testing/trino-testing-services/src/main/java/io/trino/testing/services/junit/LogTestDurationListener.java

```diff
@@ -42,15 +42,13 @@
 import static java.lang.management.ManagementFactory.getThreadMXBean;
 import static java.util.concurrent.Executors.newSingleThreadScheduledExecutor;
 import static java.util.concurrent.TimeUnit.MINUTES;
-import static java.util.concurrent.TimeUnit.SECONDS;
 import static java.util.stream.Collectors.joining;
 
 public class LogTestDurationListener
         implements TestExecutionListener
 {
     private static final Logger log = Logger.get(LogTestDurationListener.class);
 
-    private static final Duration TEST_LOGGING_THRESHOLD = new Duration(20, SECONDS);
```

Comment:
why not change the threshold instead? It helps identify failures on CI when for example a test triggers GC or runs too long.

Reply:
Junit logs the time a test took to run, so you end up with two log lines for every test

Reply:
where? I don't see it for example in https://github.com/trinodb/trino/actions/runs/8292705737/job/22697513919

Surefire logs the time per test class which is not useful enough.